ci/qa/behat

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+
+cd $(dirname $0)/../../
+
+vendor/bin/behat --stop-on-failure --config ./ci/qa/qa-config/behat.yml --tags=~skip
+echo "Behat tests passed"
```

Comment:
I personally prefer a pre run statement (with newlines to visually distinguish the output of the previous run with that of the current output. 

For that you need to use `printf` as `echo` does not allow adding linefeeds `\n`

This feedback also goes for the other places where you added a 'post-run' report.

Reply:
echo has been replaced by printf where it was used, I will remember this for the next time i need to ouput text.

Reply:
And please printf before running the test. Like we report on the other tasks (we advertise a task is about to run).

Reply:
I have changed it into a start message instead of a completed message.